src/Sulu/Bundle/MarkupBundle/DependencyInjection/CompilerPass/ParserCompilerPass.php

```diff
@@ -20,29 +20,34 @@
  */
 class ParserCompilerPass implements CompilerPassInterface
 {
-    const SERVICE_ID = 'sulu_markup.response_listener';
+    const SERVICE_IDS = [
+        'sulu_markup.response_listener',
+        'sulu_markup.swift_mailer_listener'
```

Comment:
for new services we should use something like here: https://github.com/sulu/SuluHeadlessBundle/pull/20/files#diff-66e8662ac6150c33e219664495191cf6R105

so no changes in this compilerpass should be needed